src/main.rs

```diff
@@ -36,23 +41,28 @@ fn publish(docs_path: &str,
            sign: bool,
            push_remote: &str,
            push_branch: &str,
+           shell: &mut MultiShell,
            dry_run: bool)
            -> Result<bool, FatalError> {
-    println!("Publishing Sphinx docs to GitHub Pages.");
+    try!(shell.verbose(|s| s.say("Publishing Sphinx docs to GitHub Pages.", color::BLUE)));
     let docs_build_path = format!("{}/_build/html", docs_path);
 
-    try!(git::init(&docs_build_path, dry_run));
-    try!(git::add_all(&docs_build_path, dry_run));
-    try!(git::commit_all(&docs_build_path, commit_msg, sign, dry_run));
+    try!(git::init(&docs_build_path, shell, dry_run));
+    try!(git::add_all(&docs_build_path, shell, dry_run));
+    try!(git::commit_all(&docs_build_path, commit_msg, sign, shell, dry_run));
     let remote = try!(git::remote_get_url(push_remote));
 
     let mut refspec = String::from("master:");
     refspec.push_str(push_branch);
 
-    git::force_push(docs_path, remote.trim(), &refspec, dry_run)
+    git::force_push(docs_path, remote.trim(), &refspec, shell, dry_run)
 }
 
-fn execute(args: &ArgMatches) -> Result<i32, FatalError> {
+fn execute(args: &ArgMatches, cargo_config: &CargoConfig) -> Result<i32, FatalError> {
+    try!(cargo_config.configure_shell(args.occurrences_of("verbose") as u32,
+                                      Some(args.is_present("quiet")),
+                                      &args.value_of("color").map(|s| String::from(s))));
```

Comment:
I don't think so. I'd even think it would be slightly inconsistent with the other cargo plugins as those are standard options and no one else has moved them to the `toml` as far as I'm aware.